package.json

```
@@ -15,10 +15,17 @@
         "url": "https://github.com/ara-lang/highlighting"
     },
     "license": "(Apache-2.0 OR MIT)",
-    "author": {
-        "name": "Nicolas Hedger",
-        "email": "nicolas@hedger.ch"
-    },
+    "contributors": [
+        {
+            "name": "Nicolas Hedger",
+            "email": "nicolas@hedger.ch",
+            "url": "https://github.com/nhedger"
+        },
+        {
+            "name": "Saif Eddin Gmati",
+            "url": "https://github.com/azjezz"
+        }
+    ],
```

```suggestion
    "contributors": [
        {
            "name": "Nicolas Hedger",
            "email": "nicolas@hedger.ch",
            "url": "https://github.com/nhedger"
        },
        {
            "name": "Saif Eddin Gmati",
            "email": "azjezz@protonmail.com",
            "url": "https://github.com/azjezz"
        }
    ],
```

please include a link to all contributors on the repo as well like we do in the readme.